pkg/kubelet/pod_workers.go

```diff
@@ -943,11 +943,7 @@ func (p *podWorkers) UpdatePod(options UpdatePodOptions) {
 
 		// spawn a pod worker
 		go func() {
-			// TODO: this should be a wait.Until with backoff to handle panics, and
-			// accept a context for shutdown
-			defer runtime.HandleCrash()
```

Comment:
From the first look removing this contradicts with the comment about handling panics. Can you explain how the new code will handle panics?

Reply:
> From the first look removing this contradicts with the comment about handling panics. Can you explain how the new code will handle panics?

It already exists in wait.Until.